test/src/unit-capi-attributes.cc

```diff
@@ -364,6 +367,86 @@ TEST_CASE_METHOD(
     tiledb_array_free(&array);
     tiledb_query_free(&query);
 
+    // Evolve schema BLOB -> WKB
```

Comment:
What we want here is slightly different. We want to create attribute "a" as a type blob. Write to it. Then drop "a", then add "a" again, but as TILEDB_GEOM_WKB. Then you should be able to read the data of "a" that was written as a BLOB before, but now as a TILEDB_GEOM_WKB. This test case can be added separately if it's not trivial.